files/default.conf

```diff
@@ -6,10 +6,41 @@ server {
 
     server_name _;
 
-    include sites-available/redirections.conf;
+    location / {
+        # This should never happen! Don't redirect permanently because we don't
+        # want browsers caching whatever mistake landed us here
+        rewrite_log on;
+        rewrite ^ http://www.kobotoolbox.org;
+    }
+}
+
+server {
+    server_name help.kobotoolbox.org;
+    listen 80;
+    listen [::]:80;
+    root /var/www/html;
+    index index.html;
 
     location / {
-        return 204;
+        # Requests to the base hostname now go to the Discourse forum.
+        # Note that ^$ doesn't cut it: we have to allow for the slash.
+        rewrite ^/?$ http://community.kobotoolbox.org permanent;
+
+        # Requests for anything else go to Intercom's new location
+        rewrite ^ http://support.kobotoolbox.org$request_uri? permanent;
```

Comment:
Even if it works, I've read somewhere it's now recommended to use `return` instead of `rewrite` for nginx rules. 
I think it should look something like that.
```
# Requests to the base hostname now go to the Discourse forum.
location = / {
    return 301 http://community.kobotoolbox.org;
}

# Requests for anything else go to Intercom's new location
location / {
    return 301 http://support.kobotoolbox.org$request_uri?;
}

Reply:
What about redirecting to https? Discourse does support it.

Reply:
Forgive me for not attending to this in over a year :blush: FWIW, this branch has been running the whole time :innocent: 

Regarding `return`, I guess the comments weren't clear enough. `help.kobotoolbox.org<optional forward slash><full stop>` goes to `community.kobotoolbox.org`, while `help.kobotoolbox.org/something-anything` goes to `support.kobotoolbox.org/something-anything`.

I think the first `rewrite` has to stay; if it were changed to `return`, wouldn't the second rule never be considered? I'll change the second `rewrite` to a `return`. I'll also use `https://community.kobotoolbox.org` instead of plain HTTP.